theme/base-2016/extrafields.twig

```diff
@@ -4,7 +4,7 @@
 
     <h1>{{ record.title }}</h1>
 
-    {{ fields(template = 'partials/_sub_fields.twig') }}
+    {{ fields(record = record, template = 'partials/_sub_fields.twig') }}
```

Comment:
Ok so we should add a deprecation message to that function then. And throw exception if twig_globals is disabled and a record isn't passed in.